### ISSUE TITLE
How can the defaults be altered on this module

### ISSUE THREAD

**Issue by chrisivens** [olsonpm/hoverintent-jqplugin#1]
I'm struggling to set a different timeout as h.options is never returned to a scope that I can read/set externally that I can see.

**Comment by olsonpm** [comment]
Hey - I'm really busy today and tomorrow.  I'll set some time aside wednesday to take a look.

**Comment by olsonpm** [comment]
Okay, so I looked at the code and I need to update it to [version 1.8](http://cherne.net/brian/resources/jquery.hoverIntent-1.8.0.js).  I didn't realize that I also mis-named hoverintent, where it should have a capitol I as 'hoverIntent'.  Considering semver, that will be a breaking API change so I'll have to release it as version 2.0.

I apologize for this - I started this library before I was familiar with common conventions and javascript as a whole.  I will plan on getting a fix out this weekend.  I will also add a test to ensure options are set appropriately.

Thanks for pointing this issue out
